lib/ueberauth/strategy/auth0.ex

```diff
@@ -9,17 +9,17 @@ defmodule Ueberauth.Strategy.Auth0 do
         providers: [
           auth0: { Ueberauth.Strategy.Auth0, [uid_field: :email] }
         ]
-  Default is `:user_id`
+  Default is `:sub`
 
   To set the default ['scopes'](https://auth0.com/docs/scopes) (permissions):
       config :ueberauth, Ueberauth,
         providers: [
-          auth0: { Ueberauth.Strategy.Auth0, [default_scope: "openid email"] }
+          auth0: { Ueberauth.Strategy.Auth0, [default_scope: "openid profile email"] }
         ]
-  Deafult is "openid email"
+  Deafult is `"openid profile email"`
   """
-  use Ueberauth.Strategy, uid_field: :user_id,
-                          default_scope: "openid email",
+  use Ueberauth.Strategy, uid_field: :sub,
```

Comment:
Was this a recent change with Auth0?

Reply:
Nope, but `:user_id` is just missing.

Identity user:

```
%{"email" => "test@wemake.services", "email_verified" => false,
  "name" => "test@wemake.services", "nickname" => "test",
  "picture" => "https://s.gravatar.com/avatar/3b2885e3c1ca5a7b6075d6e303437bd6?s=480&r=pg&d=https%3A%2F%2Fcdn.auth0.com%2Favatars%2Fte.png",
  "sub" => "auth0|59a163...ca83c9585",
  "updated_at" => "2017-08-26T15:19:20.232Z"}
```

Social user:

```
%{"email" => "mail@sobolevn.me", "name" => "Nikita Sobolev",
  "nickname" => "sobolevn",
  "picture" => "https://avatars1.githubusercontent.com/u/4660275?v=4",
  "sub" => "github|4660275", "updated_at" => "2017-08-26T15:18:57.497Z"}
```

There are no rules or hooks configured for the app.